tests/system/test_gbq.py

```diff
@@ -474,11 +474,14 @@ def test_timeout_configuration(self, project_id):
         select count(*) from unnest(generate_array(1,1000000)), unnest(generate_array(1, 10000))
         """
         configs = [
+            # we have a minimum limit on the timeout_ms being 400 milliseconds
+            # see pandas-gbq/gbq.py/GbqConnector/run_query docstring
+            # for more details.
             # pandas-gbq timeout configuration. Transformed to REST API compatible version.
-            {"query": {"useQueryCache": False, "timeoutMs": 1}},
+            {"query": {"useQueryCache": False, "timeoutMs": 401}},
             # REST API job timeout. See:
             # https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfiguration.FIELDS.job_timeout_ms
-            {"query": {"useQueryCache": False}, "jobTimeoutMs": 1},
+            {"query": {"useQueryCache": False}, "jobTimeoutMs": 401},
```

Comment:
Shall we add a test case where `timeout <= 400` and we verify that an exception is raised?

Reply:
Lemme take a look.

Reply:
Done.